File: 283_Move zeros/sol.cpp
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
      int cur=0;
      for(int i=0;i<nums.size();i++){
        if(nums[i]!=0){
          swap(nums[i],nums[cur]);
          cur++;
        }
      }
  }
};

File: 283_Move zeros/sol.py
class Solution(object):
    def moveZeroes(self, nums):
        """
        :type nums: List[int]
        :rtype: void Do not return anything, modify nums in-place instead.
        """
        for n in nums:
            if n == 0:
                nums.remove(n)
                nums.append(0)
